ec2utils/ec2uploadimg/lib/ec2utils/ec2uploadimg.py

```diff
@@ -18,7 +18,9 @@
 import boto3
 import os
 import paramiko
+import string
 import sys
+from tempfile import mkstemp
```

Comment:
Style nitpick, please move "from" form of import after the "import"

Reply:
Okay, done.